hummingbot/strategy/pure_market_making/pure_market_making.pyx

```diff
@@ -604,20 +605,36 @@ cdef class PureMarketMakingStrategy(StrategyBase):
             MarketBase market = self._market_info.market
             list buys = []
             list sells = []
-        for level in range(0, self._buy_levels):
-            price = self.c_get_mid_price() * (Decimal("1") - self._bid_spread - (level * self._order_level_spread))
-            price = market.c_quantize_order_price(self.trading_pair, price)
-            size = self._order_amount + (self._order_level_amount * level)
-            size = market.c_quantize_order_amount(self.trading_pair, size)
-            if size > 0:
-                buys.append(PriceSize(price, size))
-        for level in range(0, self._sell_levels):
-            price = self.c_get_mid_price() * (Decimal("1") + self._ask_spread + (level * self._order_level_spread))
-            price = market.c_quantize_order_price(self.trading_pair, price)
-            size = self._order_amount + (self._order_level_amount * level)
-            size = market.c_quantize_order_amount(self.trading_pair, size)
-            if size > 0:
-                sells.append(PriceSize(price, size))
+
+        order_override = self._order_override
+        if len(order_override) > 0:
+            for key, value in order_override.items():
+                if str(value[0]) in ['buy', 'sell']:
+                    if str(value[0]) == "buy":
+                        price = self.c_get_mid_price() * (Decimal("1") - float(value[1]))
```

Comment:
`(Decimal("1") - float(value[1]))` this will definitely give you a runtime error.